install solo5-bindings-hvt 0.4.1 fails when "tenders/hvt/solo5-hvt-debug" does not exist

I expect this check is always returning true `[ -f tenders/hvt/solo5-hvt-debug ]` is might be missing a `@`

```
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[ERROR] The installation of solo5-bindings-hvt failed at "gmake V=1 install-opam-hvt PREFIX=/home/asteen/.opam/default".

#=== ERROR while installing solo5-bindings-hvt.0.4.1 ==========================#
# context     2.0.3 | openbsd/x86_64 | ocaml-system.4.07.1 | pinned(git+https://github.com/adamsteen/solo5.git#openbsd_fix_345#b4cfd16b)
# path        ~/.opam/default/.opam-switch/build/solo5-bindings-hvt.0.4.1
# command     /usr/local/bin/gmake V=1 install-opam-hvt PREFIX=/home/asteen/.opam/default
# exit-code   2
# env-file    ~/.opam/log/solo5-bindings-hvt-55031-af57d6.env
# output-file ~/.opam/log/solo5-bindings-hvt-55031-af57d6.out
### output ###
# [...]
# mkdir -p /home/asteen/.opam/default/lib/pkgconfig \
#     /home/asteen/.opam/default/lib/solo5-bindings-hvt \
#     /home/asteen/.opam/default/include/solo5-bindings-hvt/solo5 \
#     /home/asteen/.opam/default/include/solo5-bindings-hvt/crt
# cp -R include/solo5/ include/crt/ /home/asteen/.opam/default/include/solo5-bindings-hvt
# cp bindings/hvt/solo5_hvt.o bindings/hvt/solo5_hvt.lds \
#     /home/asteen/.opam/default/lib/solo5-bindings-hvt
# cp solo5-bindings-hvt.pc /home/asteen/.opam/default/lib/pkgconfig
# cp tenders/hvt/solo5-hvt tenders/hvt/solo5-hvt-configure /home/asteen/.opam/default/bin
# [ -f tenders/hvt/solo5-hvt-debug ] && \
#     cp tenders/hvt/solo5-hvt-debug /home/asteen/.opam/default/bin
# gmake: *** [GNUmakefile:67: install-opam-hvt] Error 1



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - install solo5-bindings-hvt 0.4.1
+-
- No changes have been performed
gmake: *** [Makefile:15: depends] Error 31
```

I tried to work out a fix, but my GNU make skills are really lacking.

It needs a "-" at the start of the line, not an "@". Sorry about that.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        From: TheOtherAdamSent: Thursday, 28 March 2019 14:36To: Solo5/solo5Reply To: Solo5/solo5Cc: SubscribedSubject: [Solo5/solo5] install solo5-bindings-hvt 0.4.1 fails when "tenders/hvt/solo5-hvt-debug" does not exist (#348)I expect this check is always returning true [ -f tenders/hvt/solo5-hvt-debug ] is might be missing a @
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[ERROR] The installation of solo5-bindings-hvt failed at "gmake V=1 install-opam-hvt PREFIX=/home/asteen/.opam/default".

#=== ERROR while installing solo5-bindings-hvt.0.4.1 ==========================#
# context     2.0.3 | openbsd/x86_64 | ocaml-system.4.07.1 | pinned(git+https://github.com/adamsteen/solo5.git#openbsd_fix_345#b4cfd16b)
# path        ~/.opam/default/.opam-switch/build/solo5-bindings-hvt.0.4.1
# command     /usr/local/bin/gmake V=1 install-opam-hvt PREFIX=/home/asteen/.opam/default
# exit-code   2
# env-file    ~/.opam/log/solo5-bindings-hvt-55031-af57d6.env
# output-file ~/.opam/log/solo5-bindings-hvt-55031-af57d6.out
### output ###
# [...]
# mkdir -p /home/asteen/.opam/default/lib/pkgconfig \
#     /home/asteen/.opam/default/lib/solo5-bindings-hvt \
#     /home/asteen/.opam/default/include/solo5-bindings-hvt/solo5 \
#     /home/asteen/.opam/default/include/solo5-bindings-hvt/crt
# cp -R include/solo5/ include/crt/ /home/asteen/.opam/default/include/solo5-bindings-hvt
# cp bindings/hvt/solo5_hvt.o bindings/hvt/solo5_hvt.lds \
#     /home/asteen/.opam/default/lib/solo5-bindings-hvt
# cp solo5-bindings-hvt.pc /home/asteen/.opam/default/lib/pkgconfig
# cp tenders/hvt/solo5-hvt tenders/hvt/solo5-hvt-configure /home/asteen/.opam/default/bin
# [ -f tenders/hvt/solo5-hvt-debug ] && \
#     cp tenders/hvt/solo5-hvt-debug /home/asteen/.opam/default/bin
# gmake: *** [GNUmakefile:67: install-opam-hvt] Error 1



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - install solo5-bindings-hvt 0.4.1
+-
- No changes have been performed
gmake: *** [Makefile:15: depends] Error 31

I tried to work out a fix, but my GNU make skills are really lacking.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Solo5/solo5","title":"Solo5/solo5","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/Solo5/solo5"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"install solo5-bindings-hvt 0.4.1 fails when \"tenders/hvt/solo5-hvt-debug\" does not exist (#348)"}],"action":{"name":"View Issue","url":"https://github.com/Solo5/solo5/issues/348"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Solo5/solo5/issues/348",
"url": "https://github.com/Solo5/solo5/issues/348",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -70,7 +70,7 @@ install-opam-%: all solo5-bindings-%.pc force-install
 	cp solo5-bindings-$*.pc $(PREFIX)/lib/pkgconfig
 ifdef CONFIG_HVT
 	cp tenders/hvt/solo5-hvt tenders/hvt/solo5-hvt-configure $(PREFIX)/bin
-	[ -f tenders/hvt/solo5-hvt-debug ] && \
+	- [ -f tenders/hvt/solo5-hvt-debug ] && \
 	    cp tenders/hvt/solo5-hvt-debug $(PREFIX)/bin
 endif
 ifdef CONFIG_SPT

